build.sbt

```
@@ -95,19 +95,17 @@ lazy val effekt: CrossProject = crossProject(JSPlatform, JVMPlatform).in(file("e
     // disable tests for assembly to speed up build
     assembly / test := {},
 
-
     // Options to compile Effekt with native-image
     // -------------------------------------------
```

```suggestion
    // -------------------------------------------
    nativeImageVersion  := "22.3.0",
```
I needed to add this in order to build it on M1